Frontend/src/pages/registrations/components/RegistrationList.jsx

```diff
@@ -1,101 +1,112 @@
-import React, { useState } from 'react'
-import deleteRegistration from '../../../api/registration/delete/delete_registration'
-import getRegistrations from '../../../api/registration/get/get_registrations'
-import updateRegistration from '../../../api/registration/patch/update_registration'
+import { NonInteractiveTable } from '@thewca/wca-components'
+import React, { useEffect, useMemo, useState } from 'react'
+import { useParams } from 'react-router-dom'
+import getCompetitionInfo from '../../../api/competition/get/get_competition_info'
+import { getConfirmedRegistrations } from '../../../api/registration/get/get_registrations'
+import getCompetitorInfo from '../../../api/user/get/get_user_info'
 import styles from './list.module.scss'
-import StatusDropdown from './StatusDropdown'
 
-function RegistrationRow({
-  competitorId,
-  eventIDs,
-  serverStatus,
-  competitionID,
-  setRegistrationList,
-  registrationList,
-}) {
-  const [status, setStatus] = useState(serverStatus)
+export default function RegistrationList() {
+  const { competition_id } = useParams()
+  const [isLoading, setIsLoading] = useState(true)
```

Comment:
There's potential for confusion later since there are 2 effects fetching data, but this `isLoading` refers to only one of them.

Reply:
makes sense, I will fetch all the data in one useEffect.